packages/driver/cypress/plugins/server.js

```diff
@@ -22,6 +25,17 @@ const createApp = (port) => {
     res.end(req.method)
   })
 
+  app.use(function (req, res, next) {
```

Comment:
By default, express.js drains data and sets the `req.body` with an empty object, not the raw buffer. That's why I placed the code here. 

I thought about the 2 options:

1. Add comment why the middleware is here.
2. Remove the middleware.

I decided to go with 2 because it is created to test body buffer is generated correctly. And in usual tests, we test if the form data is parsed correctly, not the raw data is generated correctly.